### PR TITLE
adding WAL and TSM

### PR DESCRIPTION
**Pull request by beckettsean**
@rkuchan @gunnaraasen

### CODE REVIEW

**Comment by beckettsean** [comment]
partial fix for #218

File: content/influxdb/v0.12/concepts/glossary.md [hunk 1]
```diff
@@ -207,23 +207,30 @@ See [InfluxQL Functions](/influxdb/v0.12/query_language/functions/#transformatio
...
 
 * *Admin users* have `READ` and `WRITE` access to all databases and full access to administrative queries and user management commands.
 * *Non-admin users* have `READ`, `WRITE`, or `ALL` (both `READ` and `WRITE`) access per database.
 
-
 When authentication is enabled, InfluxDB only executes HTTP requests that are sent with a valid username and password.
 See [Authentication and Authorization](/influxdb/v0.12/administration/authentication_and_authorization/).
 
+## wal (Write Ahead Log) 
+The temporary cache for recently written points. To reduce the frequency with which the permanent storage files are accessed, InfluxDB caches new points in the WAL until their total size or age triggers a flush to more permanant storage. This allows for efficient batching of the writes into the TSM.
```
Comment (rkuchan) [inline content/influxdb/v0.12/concepts/glossary.md]: permanant -> permanent

**Comment by rkuchan** [comment]
The other glossary entries that have an abbreviation (like [retention policy](https://docs.influxdata.com/influxdb/v0.12/concepts/glossary/#retention-policy-rp)) write out the full word and then put the abbreviation in parentheses. It's not a big deal but it might be nice to have a consistent style. 

Otherwise 👍

**Comment by beckettsean** [comment]
I think since `TSM` and `WAL` are almost always used as acronyms it makes sense for the acronym to come first. That way deep links are http://docs.influxdata.com/influxdb/v0.12/concepts/glossary/#wal instead of http://docs.influxdata.com/influxdb/v0.12/concepts/glossary/#write-ahead-log.

I am about 53/47 in favor of leading with the acronym but I'm happy to revert if you prefer.

**Comment by beckettsean** [comment]
@gunnaraasen can you tie-break for us on the entry titles? 

 - WAL (Write Ahead Log)

OR

- Write Ahead Log (WAL)

**Comment by gunnaraasen** [comment]
I'm not really partial to one or the other, but I'll go with the acronym first since that's how we colloquially refer to the TSM and WAL.

**Pull request merged**